salt/modules/cmdmod.py

```diff
@@ -725,13 +748,13 @@ def _run_all_quiet(cmd,
                    saltenv='base',
                    pillarenv=None,
                    pillar_override=None,
-                   output_loglevel=None):
+                   output_encoding=None):
```

Comment:
NOTE: This is intentional, not a mistake. We don't do anything with output_loglevel, since we're just passing it as `'quiet'` when we invoke `_run()`.